prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -136,8 +226,13 @@ def smart_charging(
         charging_efficiency,
     )
 
+    output_load_sum_list = []
+
     day_num = len(input_day)
     for day_iter in range(day_num):
+        print(f"Day: {day_iter}")
```

Comment:
Should this output controlled by `debug_printout` as well, or we simply wanna track the progress of the loop, in which case `tdqm` can be used as elsewhere in the code base.

Reply:
This we would like to keep as a progress tracker for someone running the code.

Reply:
> This we would like to keep as a progress tracker for someone running the code.

Then we could use a `tqdm` to generate a progress bar instead of printing things out iteratively like [here](https://github.com/Breakthrough-Energy/PreREISE/blob/53fed977feced25bda68b49a4a4a6aa8e5c90626/prereise/gather/demanddata/eia/map_ba.py#L90)